package.json

```diff
@@ -25,15 +25,17 @@
     "eslint-plugin-react": "^7.14.2",
     "eslint-plugin-standard": "^4.0.0",
     "ethlint": "^1.2.4",
+    "openzeppelin-solidity": "2.0.1",
     "prettier": "^1.18.2",
-    "solidity-coverage": "^0.6.2"
+    "solidity-coverage": "^0.6.2",
+    "truffle": "^4.1.14"
   },
   "scripts": {
     "start": "npm run start:ipfs",
```

Comment:
We could set this to `npm run start:ipfs:template`, right?

Reply:
Ah, yep. We could also not specify ipfs and let it be the default.

So it would be 

```
   "start": "npm run start:template",
    "start:http": "npm run start:http:template",
    "start:template": "./scripts/start.sh ipfs",
    "start:http:template": "./scripts/start.sh http",
```

What do you think?

Reply:
No, I would preserve the "standard" way as much as I can (so other Aragon devs can onboard as easy as possible). The only change I would do is for `npm start`, since the default one is not useful because it does not set up the token manager and the vault.